test/functional/rawtransactions.py

```diff
@@ -39,6 +49,41 @@ def run_test(self):
         self.nodes[0].generate(5)
         self.sync_all()
 
+        # Test `createrawtransaction` required parameters
+        assert_raises_rpc_error(-1, "createrawtransaction", self.nodes[0].createrawtransaction)
+        assert_raises_rpc_error(-1, "createrawtransaction", self.nodes[0].createrawtransaction, [])
+
+        # Test `createrawtransaction` invalid extra parameters
+        assert_raises_rpc_error(-1, "createrawtransaction", self.nodes[0].createrawtransaction, [], {}, 0, False, 'foo')
```

Comment:
nit: My personal style preference is to use named args when passing unnamed parameters. Not sure what others think.

Reply:
In this case I'm not sure, because that will throw `Unknown named parameter`. But for other cases you mean?
```py
assert_raises_rpc_error(-1, "createrawtransaction", self.nodes[0].createrawtransaction, inputs=[], outputs={}, locktime=0, replaceable=False)
```

Reply:
Ah, right. Won't work in this case.